stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -456,6 +456,12 @@ function get_type(sym::Expr, fn::Module)
     # try to analyze nests of calls. if this fails, try using the expanded form.
     val, found = try_get_type(sym, fn)
     found && return val, found
+    # https://github.com/JuliaLang/julia/issues/27184
+    if isexpr(sym, :macrocall)
+        s = first(sym.args)
+        isa(s, Symbol) && !isdefined(fn, s) && return Any, false
+        isa(s, GlobalRef) && !isdefined(s.mod, s.name) && return Any, false
```

Comment:
This does not seem to handle `A.@b. <tab>` yet

Reply:
ops, thanks !